scilpy/tracking/local_tracking.py

```diff
@@ -0,0 +1,365 @@
+# -*- coding: utf-8 -*-
```

Comment:
Here could you replace print by logging.debug or logging.error.

Reply:
Done in 0c68e20